kyuubi-main/src/main/scala/org/apache/kyuubi/engine/SQLEngineAppName.scala

```diff
@@ -32,6 +32,13 @@ case class SQLEngineAppName(
       case _ => ZKPaths.makePath(s"${prefix}_$sharedLevel", user)
     }
   }
+
+  def getZkLockPath(prefix: String): String = {
+    assert(sharedLevel != CONNECTION)
+    sharedLevel match {
+      case _ => ZKPaths.makePath(s"${prefix}_$sharedLevel", "lock", user)
```

Comment:
`ZKPaths.makePath(s"${prefix}_$sharedLevel", "lock", user) `？

Reply:
Hi @Honglun didn't you miss this comment? why do we need to match with _ here?

Reply:
I see, I will update this.